pandas/core/indexes/base.py

```diff
@@ -395,7 +392,6 @@ def _outer_indexer(
     # Whether this index is a NumericIndex, but not a Int64Index, Float64Index,
     # UInt64Index or RangeIndex. Needed for backwards compat. Remove this attribute and
     # associated code in pandas 2.0.
```

Comment:
Guessing this comment isn't needed anymore?

Reply:
Yeah…I’l wait a bit to see if there are more comments coming, then I’ll update.

Reply:
I’ve updated this now.